setup.py

```diff
@@ -53,6 +53,6 @@
     setup_requires=["cffi"],
     ext_package="metaswitch.common",
     cffi_modules=["cffi_build.py:ffi"],
-    install_requires=["py-bcrypt", "pycrypto==2.6.1", "pyzmq==15.2", "cffi==1.5.2"],
+    install_requires=["py-bcrypt", "pycrypto==2.6.1", "pyzmq==15.2", "cffi==1.5.2", "monotonic==0.6"],
```

Comment:
Does `setup.py` support SemVer pinning?  E.g. `monotonic~>0.6` to pin to `0.6.x` for any `x`.  Not useful here, since monotonic is not SemVer... but could be useful elsewhere (to pick up bug fixes without breaking things)